src/App.tsx

```diff
@@ -15,48 +17,115 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortField {
+  string = 'alphabetically',
```

Comment:
pls refactor the enum according to [enums naming convention](https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/typescript/data-types/enums.html#:~:text=Naming%20convention,enum%20tend%20to%20be%20titlecase)